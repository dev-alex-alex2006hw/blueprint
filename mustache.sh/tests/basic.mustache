{{#FOO}}foo{{/FOO}} {{BAR}} {{#BAR}}baz{{/BAR}}
{{^FOO}}
nothing to see here
{{/FOO}}
{{!comment comment { comment comment comment}}
{{#FOO}}
foo {{BAR}} baz
{{/FOO}}
{{#FOO}}{{#BAR}}{{^BAZ}}nested{nested{{/BAZ}}{{/BAR}}{{/FOO}}
