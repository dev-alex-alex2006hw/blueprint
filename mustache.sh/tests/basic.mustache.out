foo bar baz



foo bar baz

nested{nested
